src/main/java/com/tarento/assetManagement/dao/model/Asset.java

```diff
@@ -0,0 +1,133 @@
+package com.tarento.assetManagement.dao.model;
+
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Date;
+
+@Entity
+@Table(name = "assetDetails",schema = "assetManagement")
+public class Asset implements Serializable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
```

Comment:
do we need @Column attribute now , i feel you can remove it.

Reply:
Will try without column and check Manzarul